modules/settings.js

```diff
@@ -38,10 +37,29 @@ Settings.prototype.initialize = function() {
     // If load failed, the local storage "options" object
     // was malformed. It has now been cleared and Settings
     // options holds default values.
+    this.set(this.getDefault());
   }
 };
 
 /**
+ * @access public
+ * @description Returns settings default values.
+ */
+
+Settings.prototype.getDefault = function() {
```

Comment:
Music hasn't been tested yet, so it's disabled for now. Might not be a bad thing to include though